db/models.py

```diff
@@ -14,3 +14,39 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
+    genres = models.ManyToManyField(Genre, related_name="genres")
```

Comment:
The `related_name` in the `ManyToManyField` is incorrect. It should represent the name of the model in the reverse relation. In this case, it should be 'movies' instead of 'genres'. The `related_name` option creates a reverse relation from the `Genre` model to the `Movie` model. So it should be something that makes sense when accessed from a `Genre` object. Here is the corrected code: `genres = models.ManyToManyField(Genre, related_name='movies')`.